zerver/lib/documentation_redirects.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class DocRedirect:
+    old_url: str
+    new_url: str
+
+
+""" To add a new redirect to the list, follow these formats:
+
+    REST API documentation:
+    DocRedirect("api/old", "/api/new"),
+
+    Help center documentation:
+    DocRedirect("help/old", "/help/new"),
+
+    Note that the old URL should not start with a forward slash,
+    while the new URL should.
```

Comment:
It could be helpful to couple checks like `not assert old_url.startswith("/")` inside the loop in `zproject/urls.py` to catch this class of error.

Reply:
Decided to go with the latter option, so the forward slash is stripped from the string in `zproject.urls.py`, which makes the format for the old and new URLs the same.